src/renderer/features/operations/OperationsValidation/lib/transfer-rules.ts

```diff
@@ -91,10 +91,12 @@ export const TransferRules = {
       validator: (
         amount: string,
         _: any,
-        { network, balance }: { network: NetworkStore | null; balance: BalanceMap },
+        { network, balance }: { network: NetworkStore | null; balance?: BalanceMap },
       ) => {
         if (!network) return false;
 
+        if (!balance) return true;
```

Comment:
As i understand it, you skip validation if there is no balance. IMO it's better to fail like it's done with `networks` couple lines before